src/pages/Admin/EntranceExperience/EntranceExperience.tsx

```diff
@@ -138,7 +140,9 @@ const EntranceExperience: React.FC<EntranceExperienceProps> = ({
           </S.ItemBody>
         </S.ItemWrapper>
 
-        <Button type="submit">Save</Button>
+        <Button className="EntranceExperience__save-button" type="submit">
+          Save
+        </Button>
```

Comment:
Maybe OOS, but if there is a fast few minute ButtonNG replacement possible - FTW

Reply:
Fixed in: ea4802ee1338ca4c0bf535218a1562a1d7c45bf3